src/algorithms/ctmrg/initialization.jl

```diff
@@ -0,0 +1,56 @@
+abstract type InitializationStyle end
+struct ProductStateInitialization <: InitializationStyle end
+struct RandomInitialization <: InitializationStyle end
```

Comment:
Should this one encapsulate which initialization function we are using, so it can switch between `randn` and `rand`?

Reply:
Yes, that makes a lot of sense.